x-pack/plugins/fleet/public/applications/fleet/sections/epm/screens/detail/index.test.tsx

```diff
@@ -48,6 +49,39 @@ describe('when on integration detail', () => {
     window.location.hash = '#/';
   });
 
+  describe('and the package is installed', () => {
+    beforeEach(() => render());
+
+    it('should display agent policy usage count', async () => {
+      await mockedApi.waitForApi();
+      expect(renderResult.queryByTestId('agentPolicyCount')).not.toBeNull();
+    });
+
+    it('should the Policies tab', async () => {
```

Comment:
what does this mean

Reply:
Will correct. should be: 'should show the policies tab'